crates/webidl/src/util.rs

```diff
@@ -500,8 +501,13 @@ impl<'src> FirstPassRecord<'src> {
         }
 
         for interface in &mut ret {
-            if let Some(fixed) = FIXED_INTERFACES.get(&interface.name.to_string().as_ref()) {
-                interface.name = rust_ident(fixed);
+            if let Some(type_name) = type_name {
+                if let Some(map) = FIXED_INTERFACES.get(type_name) {
+                    if let Some(fixed) = map.get(&interface.name.to_string().as_ref())
```

Comment:
I think it'd be nicer to use `Option::and_then` rather than having these nested `if let`s, but that's just a nitpick / matter of opinion. Feel free to disregard.

Reply:
Thanks!
Couldn't merge all three because `map` is a temporary variable.